core/server/api/v2/utils/validators/input/schemas/posts.json

```diff
@@ -142,6 +142,12 @@
                     "email": {
                         "type": "string",
                         "maxLength": 191
+                    },
+                    "roles": {
```

Comment:
What is the state of "Any unknown fields should error"?

A "posts-authors" schema would be cool, but feels not important right now?

Reply:
Skipped this property validation for relations to avoid any unexpected errors. For now, it's validating required fields which is good enough. But definitely can introduce `authors` schema and build this relation based on that. But also think it's a secondary priority :+1: